lib/typeUtil.js

```diff
@@ -113,5 +113,6 @@ function _objectToValue(obj) {
 module.exports = {
   unpackObjectOrArray: unpackObjectOrArray,
   packObjectOrArray: packObjectOrArray,
-  valueToObject: valueToObject
+  valueToObject: valueToObject,
+  objectToValue: objectToValue
```

Comment:
Hmm.  I'll remove it from this PR.  It was in anticipation of needing to implement filtering in FakeDynamo which will have to come soon.